lib/query/stored.js

```diff
@@ -1,47 +1,78 @@
 const { fetch } = require('../fetch');
-const lodashPick = require('lodash/pick');
 const { httpBody } = require('../response-transforms');
 
+const convertToJsonLd = (conn, storedQuery) => {
```

Comment:
i'm not really happy with this change; i think it's the best move for not adding a breaking change, but let me know if anyone has better ideas!

some ideas:
* add new methods where args are just passed through; ie accept, content-type, and storedQuery are just passed to fetch
* get rid of these json-style methods since they're deprecated (in stardog source code)